data/src/scripts/skill_thieving/scripts/pickpocketing/pickpocket.rs2

```diff
@@ -127,4 +127,9 @@ if (%action_delay > map_clock) {
 // inv full check happens on the engine level for pick pocketing.
 // https://oldschool.runescape.wiki/w/Update:Halloween_2016, when check was added for preventing attacking when an npc is being pickpocketed
 
-~pick_pocket($data);
\ No newline at end of file
+~pick_pocket($data);
+
+%action_delay = calc(map_clock + 3);
+p_opnpc(3);
```

Comment:
Basically thieve the NPC again (this seems to be the NPC we're already interacting with, 3 seems to be thieving). Probably has no side effects in quests and stuff